templates/docker-compose/docker-compose.secrets.yml.dist

```diff
@@ -0,0 +1,10 @@
+version: '3.8'
+
+services:
+    configurator:
+        image: public.ecr.aws/supportpal/helpdesk-configurator:latest
```

Comment:
Should we remove `CONFIGURATOR_VERSION` from `Makefile.dist`?

Reply:
I think I'll revert to use the version here, feels better suited on the longterm.

Reply:
I've changed it.